app/views/candidate_mailer/new_referee_request.text.erb

```diff
@@ -1,17 +1,9 @@
-<% if @reason == :email_bounced %>
-Your referee request did not reach <%= @reference.name %>.
+<% if @application_form.show_new_reference_flow? %>
 
-Check you gave the correct email address and request the reference again.
-<% elsif @reason == :refused %>
-<%= @reference.name %> has declined your reference request.
-<% else %>
-<%= @reference.name %> has not responded yet.
-<% end %>
-
-You can add as many referees as you like to increase the chances of getting 2 references quickly.
+  <%= render 'new_flow_new_referee_request' %>
 
-You can also send one reminder to each referee from your account.
+<% else %>
 
-[Manage your references](<%= candidate_magic_link(@candidate) %>).
+  <%= render 'old_flow_new_referee_request' %>
```

Comment:
Might need to drop the leading whitespace here too?

```suggestion
<%= render 'old_flow_new_referee_request' %>
```

Reply:
👍  removed